.circleci/config.yml

```diff
@@ -0,0 +1,246 @@
+version: 2
+
+# ################## WORKFLOWS ##################
```

Comment:
nit: I don't this is needed either, also clear that this is the workflows section